clang/test/CheckedCRewriter/checkedregionvariadic.c

```diff
@@ -0,0 +1,10 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+#include <stdio_checked.h>
```

Comment:
To include a standard header you need to add to the `CPATH` , right? @sroy4899 worked on tests like this, and I recall there being `CPATH` added to the test description.

Reply:
Yeah, the relevant `RUN` command here is:
```
// RUN: CPATH=$CHECKED_CPATH:$CPATH cconv-standalone %s -- | FileCheck -match-full-lines %s
```
For some context, @aaronjeline, take a look at `extstructfields.c`
Make sure your `lit.local.cfg` has all the right paths too.